docs/source/performance.rst

```diff
@@ -14,272 +15,228 @@ Version |release| for version |version| of the CF conventions.
+contribute for the final result.
 
-----
-
-**Regridding**
---------------
-
-When regridding multiple field constructs with their
-`cf.Field.regrids` or `cf.Field.regridc` methods, the regridding
-weights are calculated for each field's operation, and these
-calculations are often the main expense of the operation, sometimes
-close to 100% of the cost.
-
-This can be avoided for cases when all of the fields are being
-regridded to the same destination grid, and all share a common source
-grid. In such cases, the regridding operator that defines the grids
-and the weights can be calculated once and then used to regrid each
-field construct, giving potentially very large performance
-improvements.
-
-.. code-block:: python
-   :caption: *Regrid every field construct in the field list 'fl' to a
-             common destination grid defined by the field construct
-             'dst'.*
-		
-   >>> operator = fl[0].regrids(dst, method='conservative',
-   ...                          return_operator=True)
-   >>> fl_regridded = cf.FieldList(f.regrids(operator) for f in fl)
-   
+The performance of `cf` is largely a function of the performance of
+Dask. All of the techniques that Dask supports for `improving
+performance <https://docs.dask.org/en/stable/best-practices.html>`_
+apply, and performance parameters are generally be set via Dask's
+`configuration settings
+<https://docs.dask.org/en/stable/configuration.html>`_, but the value
+of the important :ref:`chunk size <Chunks>` is set through the `cf`
+API.
 
 ----
 
-.. _LAMA:
-
-**Large Amounts of Massive Arrays (LAMA)**
-------------------------------------------
-
-Data are stored and manipulated in a very memory efficient manner such
-that large numbers of constructs may co-exist and be manipulated
-regardless of the size of their data arrays. The limiting factor on
-array size is not the amount of available physical memory, but the
-amount of disk space available, which is generally far greater.
-
-The basic functionality is:
-
-* **Arrays larger than the available physical memory may be created.**
-
-  Arrays larger than a preset number of bytes are partitioned into
-  smaller sub-arrays which are not kept in memory but stored on disk,
-  either as part of the original file they were read in from or as
-  newly created temporary files, whichever is appropriate.  Therefore
-  data arrays larger than a preset size need never wholly exist in
-  memory.
+.. _Lazy-operations:
 
-* **Large numbers of arrays which are in total larger than the
-  available physical memory may co-exist.**
 
-  Large numbers of data arrays which are collectively larger than the
-  available physical memory may co-exist, regardless of whether any
-  individual array is in memory or stored on disk.
+**Lazy operations**
+-------------------
 
-* **Arrays larger than the available physical memory may be operated
-  on.**
+In general, all `cf` data operations (such as reading from disk,
```

Comment:
Is 'reading from disk' something we should list here at this point, since we state later that netCDF is read serially for now? I'd drop it (I also wouldn't classify reading it as a data operation if that sways it, but that's just me being particular):

```suggestion
In general, all `cf` data operations (such as
```

Reply:
Hi Sadie - I think I'd rather keep this. The read is lazy - the fact that the computation of the read operation happens serially is a separate performance issue - one that you'll never encounter if you don't look at the data :)